README.md

```diff
@@ -110,14 +105,48 @@ example](examples/pom_file_generation/) for more information.
 
 ## Advanced usage
 
+### Fetch source JARs
+
+To download the source JAR alongside the main artifact JAR, set `fetch_sources =
+True` in `maven_install`:
+
+```python
+maven_install(
+    artifacts = [
+        # ...
+    ],
+    repositories = [
+        # ...
+    ],
+    fetch_sources = True,
+)
+```
+
+### Checksum verification
+
+Artifact resolution will fail if a `SHA-1` or `MD5` checksum file for the
+artifact is missing in the repository. To disable this behavior, set
+`fail_on_missing_checksum = False` in `maven_install`:
+
+```python
+maven_install(
+    artifacts = [
+        # ...
+    ],
+    repositories = [
+        # ...
+    ],
+    fail_on_missing_checksum = False,
```

Comment:
Isn't the default false?

Reply:
It is not. Shouldn't it be? Oh wait it's maybe coming back to me. Does it not do anything if there isn't actually a checksum?

Reply:
It fails the build if there isn't a checksum. Why do you think it shouldn't fail if the checksum is missing?

Reply:
Err, maybe I'm misremembering how this works. If `fail_on_missing_checksum` is true, then the build should indeed fail with missing checksums :) 

but I think people should be able to write: 
```
maven_install(
    artifacts = [...],
    repositories = [...]
)
``` 

and that's it. That requires the default of `fail_on_missing_checksum` to be False. If it's not, then in order to not specify checksums for artifacts, users must write:

```
maven_install(
    artifacts = [...],
    repositories = [...],
    fail_on_missing_checksum = False,
)
``` 
which if the second snippet I wrote is the minimum, then the README should actually keep the `fail_on_misisng_checksum` attribute in this "default" snippet

Reply:
Do you mean you'd like to change the default for fail_on_missing_checksum from False to True in this PR?

Reply:
No :)
But I'm saying that 
1. maybe we should consider the default and 
2. since the default is True right now, have the example in the README on line 54ish set `fail_on_missing_checksum=False` because otherwise it's not a working example